x/oracle/keeper/oracle.go

```diff
@@ -54,6 +54,13 @@ func (k Keeper) ApproveOracleRegistration(ctx sdk.Context, msg *types.MsgApprove
 		return sdkerrors.Wrapf(types.ErrApproveOracleRegistration, err.Error())
 	}
 
+	oracleRegistration.EncryptedOraclePrivKey = msg.ApproveOracleRegistration.EncryptedOraclePrivKey
```

Comment:
Let's check if the `msg.ApproveOracleRegistration.EncryptedOraclePrivKey` is not nil in `k.validateApproveOracleRegistration()`.

Reply:
Does it mean if the `msg.ApproveOracleRegistration.EncryptedOraclePrivKey` is nil, return error?

Reply:
Applied in [3de7115](https://github.com/medibloc/panacea-core/pull/546/commits/3de71155e8d90a4b124ae319bbeefe544c84a72b).